### PR TITLE
[issues] ButtonCounter fixes

### PR DESCRIPTION
**Pull request by philli-m**

### CODE REVIEW

**Review by khamui — APPROVED** [review]
Looking good!

**Comment by khamui** [comment]
@philli-m  just realised now that on the list the buttons look quite big and vertically off (compared to the design). Should have brought it up earlier, sorry.

Bildschirm­foto 2023-03-30 um 08 51 27

**Comment by philli-m** [comment]
@khamui this should be better

File: src/theme/fonts.js [hunk 1]
```diff
@@ -24,7 +24,3 @@ export const LINEHEIGHTS = {
   lg: pixelRatio * 28,
   xl: pixelRatio * 30,
 }
-
-export const LETTERSPACING = {
-  iconbtn: 5,
-}
```
Comment (philli-m) [inline src/theme/fonts.js:30]: this didn't work when the numbers were double figures

**Comment by khamui** [comment]
really nice!

**Pull request merged**